src/libp2p/Libp2p.Protocols.Tls/TlsProtocols.cs

```diff
@@ -0,0 +1,178 @@
+using System.Buffers;
```

Comment:
Could you name it TlsProtocol instead of TlsProtocols

Reply:
done